Assets/Treasure/TDK/Runtime/Connect/TDK.Connect.cs

```diff
@@ -209,6 +206,17 @@ public async Task ConnectSocial(SocialAuthProvider provider)
 #endif
         }
 
+        public async Task Reconnect(WalletConnection lastWalletConnection)
```

Comment:
How can this be used outside of this file? It's unclear to me how any consumer would have a `WalletConnection` instance that they can reuse here

Reply:
Good point, I made it private and renamed to `ReconnectInternal`, and exposed a more user-friendly `Reconnect(email)` method